__tests__/app.test.js

```diff
@@ -315,3 +315,111 @@ describe("GET /api/users", () => {
       });
   });
 });
+
+describe("GET /api/articles sort by queries", () => {
+  test("200:should return sorted article created_by", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles).toHaveLength(11);
+        expect(body.articles).toBeSortedBy("created_at", { descending: true });
```

Comment:
For this test, could we also just check that the returned objects have the topic of 'mitch'?

Reply:
okay